### PR TITLE
add eslint config to repo to integrate with text editors

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

**Comment by origamiserviceuser** [comment]
o-errors bundle size difference from 4.0.2 to 477fa359f01190064205bc9b115e7bb201e64b2e
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-errors bundle size difference from 4.0.2 to e538771fa278d8edf3901f7fa2d2ab65c8496fb9
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-errors bundle size difference from 4.0.2 to a57852c39d6e9f0fa3df1c94a018eb3d40c02a13
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-errors bundle size difference from 4.0.2 to fe2d345afeba575294421aac5f5fdf00e7492237
No significant bundle size differences found.

File: src/js/oErrors.js [hunk 1]
```diff
@@ -462,7 +462,7 @@ Errors.prototype._updatePayloadBeforeSend = function(data) {
  * @returns {boolean} - Boolean value indicating if there's declarative config
  */
 Errors.prototype._hasDeclarativeConfig = function() {
-	return !!this._getDeclarativeConfig();
+	return Boolean(this._getDeclarativeConfig());
```
Comment (notlee) [inline src/js/oErrors.js]: 👍 nice

**Comment by origamiserviceuser** [comment]
o-errors bundle size difference from 4.0.2 to 73f816486ef7ad1dc1e5a888e21f4535fe238605
No significant bundle size differences found.

**Pull request merged**